server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java

```diff
@@ -368,27 +368,23 @@ private JettyServerConfig(Map<String, String> configs) {
 
     this.keyStoreType = internalConfig.get(SSL_KEYSTORE_TYPE);
     this.trustStoreType = internalConfig.get(SSL_TRUST_STORE_TYPE);
-    String keyStorePath = null;
-    String keyStorePassword = null;
-    String managerPassword = null;
-    String trustStorePath = null;
-    String trustStorePassword = null;
 
     if (this.enableHttps) {
-      keyStorePath = internalConfig.get(SSL_KEYSTORE_PATH);
-      keyStorePassword = internalConfig.get(SSL_KEYSTORE_PASSWORD);
-      managerPassword = internalConfig.get(SSL_MANAGER_PASSWORD);
-      if (this.enableClientAuth) {
-        trustStorePath = internalConfig.get(SSL_TRUST_STORE_PATH);
-        trustStorePassword = internalConfig.get(SSL_TRUST_STORE_PASSWORD);
-      }
+      this.keyStorePath = internalConfig.get(SSL_KEYSTORE_PATH);
+      this.keyStorePassword = internalConfig.get(SSL_KEYSTORE_PASSWORD);
+      this.managerPassword = internalConfig.get(SSL_MANAGER_PASSWORD);
+    } else {
+      this.keyStorePath = null;
+      this.keyStorePassword = null;
+      this.managerPassword = null;
+    }
+    if (this.enableHttps && this.enableClientAuth) {
+      this.trustStorePath = internalConfig.get(SSL_TRUST_STORE_PATH);
+      this.trustStorePassword = internalConfig.get(SSL_TRUST_STORE_PASSWORD);
+    } else {
+      this.trustStorePath = null;
+      this.trustStorePassword = null;
```

Comment:
Given the local variables no longer exist, there is probably no need to use "this." While we are there, we might remove unneeded "this." from other places in that method. Also, minThreads/maxThreads exist as local variables; you might consider renaming those.

Reply:
In the constructor method, we use the format `this.field`. It's code style of our project. If we decide to change this, maybe we will many places in other classes.

Reply:
@justinmclean  I think _minThreads/maxThreads_ may be reassigned and log print more information, which is why local variables are used.

Reply:
Nothing to do with this PR @qqqttt123 but I don't see this. It is inconsistantly used (it is used when paramaters of the same name are passed into the constructor which is needed), it's not documented anywhere and we have no check for it.

Reply:
Emm ... Maybe it's inconsistent. But I follow the style and the code which I read also follows the style. Maybe it's optional.

Reply:
So i think both the style and min/maxThreads are ok.
Is there anything else I need to do for this PR?  thx.

Reply:
Or @qqqttt123, even better and if possible, use Spotless to enforce this.

Reply:
> Or @qqqttt123, even better and if possible, use Spotless to enforce this.

It's an optional rule. Just for me, there are many places to follow it. I just feel that we don't need to change it.

Reply:
Even if it is optional, it should be documented.

Reply:
> Even if it is optional, it should be documented.

Make sense.